src/main/java/bio/terra/common/flagsmith/FlagsmithService.java

```diff
@@ -41,7 +41,7 @@ public class FlagsmithService {
    * @param feature the name of the feature
    */
   public Optional<Boolean> isFeatureEnabled(String feature) {
-    return isFeatureEnabled(feature, /*userEmail=*/ null);
+    return isFeatureEnabled(feature, /* userEmail= */ null);
```

Comment:
Interesting, did spotless change their rules? Anyway I would delete these comments rather than reformat, they don't seem to add value IMO. But maybe others really like them?

Reply:
Thanks for reviewing!

Yes, spotless changed their rules. I don't have a strong feeling about the comments, and generally prefer to just automatically reformat the files.